packages/document-service-api/src/controllers/documents.js

```diff
@@ -188,4 +191,45 @@ module.exports = {
       }
     },
   ],
+  deleteDocument: async (req, res) => {
+    const { documentId } = req.params;
+
+    const search = {
+      applicationId: req.params.applicationId,
+      id: documentId,
+    };
+
+    const doc = await Documents.findOne(search);
+
+    if (!doc) {
+      req.log.debug(search, 'Unknown document ID');
+
+      res.status(404).send({
+        message: 'Unknown document ID',
+      });
+      return;
+    }
+
+    const [err, data] = await Documents.deleteOne({ documentId })
+      .then((d) => [null, d])
+      .catch((e) => [e, null]);
+
+    if (err && !data) {
+      req.log.error('An error occured whilst trying to delete a document from the database.');
```

Comment:
Add the `err` object to log

`req.log.error({ err }, 'An error....')`

Suggest a more terse error message too, eg `Failed to delete document from database`